--- unoconv~	2007-09-01 04:23:13.000000000 +0300
+++ unoconv	2010-05-21 17:59:08.060940939 +0300
@@ -17,7 +17,7 @@
 import getopt, sys, os, glob, time
 
 #extrapaths = ('/usr/lib/openoffice/program/', '/usr/lib/openoffice.org2.0/program/')
-extrapaths = glob.glob('/usr/lib*/openoffice*/program/') + glob.glob('/usr/lib*/ooo*/program') + [ '/Applications/NeoOffice.app/Contents/program', ]
+extrapaths = glob.glob('/usr/lib*/openoffice*/program/') + glob.glob('/usr/lib*/openoffice*/basis*/program/')
 for path in extrapaths:
     try:
         sys.path.append(path)   
